README.md

```diff
@@ -1,222 +1,73 @@
-
-```scss
-// _variables.scss
-
-// Configure some variables for Stencil’s button:
-$button-padding: 5px 10px;
-$button-border-radius: 5px;
+grunt component:install:utils
 ```
 
-```scss
-// my-components/_button.scss
+Once installed, the Stencil Utilities live in your project’s `app/bower_components` directory. Individual utilities will provide a stylesheet, including example markup and (optionally) JavaScript. To learn more, visit the [dev centre documentaton](#).
 
-// Override or add to a Stencil style:
-.c-button {
-    border: 0; // override
-    background-color: hsl(200, 70%, 50%); // add
-}
 
-// Extend with a new modifier, in this case for a button that uses the site’s brand color:
-.c-button.c--brand-color {
-    background: $brand-color;
-    color: #fff;
-}
-```
+## Using a Stencil Utility
 
+Utility classes are conceptually a bit like a Sass mixin; they allow you to add new style rules to an existing component, except that they are applied as classes in your markup. An example utility class is `.u-width-full`. It does exactly what it says: make an element expand to fill the full width of its container.
 
-## Naming conventions
+Use these utilities in conjunction with other components for building layout (e.g. a Grid component and `.u-width-full`), or apply them to elements on your page in place of custom modifiers (e.g.: `.c-button.u-width-full` to make a button full-width without having to create a modifier).
 
-Stencil uses Mobify’s variation on the BEM (Block/Element/Modifier) convention. Read more about the details of the convention in our open source [CSS style guide](https://github.com/mobify/mobify-code-style/blob/master/css/Readme.md#component-oriented-naming).
-
-Stencil adds one extra convention to this: utility classes. These special classes are conceptually a bit like a Sass mixin: they allow you to “mix in” new styles into an existing component. Unlike mixins, Stencil utility classes are applied as classes in your markup. An example of a utility class is `u-width-full`. It does exactly what you’d think: makes an element take up the full width of its container. This utility class can be used in conjunction with the Grid component (along with other width utilities) for building layout, and can also be applied for example to make buttons full-width without needing a custom full-width button modifier.
+Once your theming is in place, and you've built any custom components you need, you'll find you can build full pages simply by writing HTML, applying utility classes to apply one-off modifications to page elements.
 
 Some things to note about utility classes:
 
 1. Utility classes should apply a focused set of CSS properties that are applicable across a wide range of components;
-2. Utility classes use a `u-` prefix instead of the `c-` prefix used for a component’s class name;
+2. Utility classes use a `.u-` prefix instead of the `.c-` prefix used for a component’s class name;
```

Comment:
Technically speaking, the period is not part of the classname prefix, I’d rather leave it off here.